src/clib/lib/job_queue/lsf_driver.cpp

```diff
@@ -472,61 +480,69 @@ static void lsf_driver_update_bjobs_table(lsf_driver_type *driver) {
     int fd = mkstemp(tmp_file);
     close(fd);
 
-    run_bjobs(driver, tmp_file);
+    int exit_code = run_bjobs(driver, tmp_file);
 
-    char status[16];
-    FILE *stream = fopen(tmp_file, "r");
-    if (!stream) {
-        throw std::runtime_error("Unable to open bjobs output: " +
-                                 std::string(strerror(errno)));
-    }
-    bool at_eof = false;
-    driver->bjobs_cache.clear();
-    skip_line(stream);
-    while (!at_eof) {
-        char *line = next_line(stream, &at_eof);
-        if (line != nullptr) {
-            int job_id_int;
-
-            if (sscanf(line, "%d %*s %s", &job_id_int, status) == 2) {
-                std::string job_id = std::to_string(job_id_int);
-                // Consider only jobs submitted by this ERT instance - not
-                // old jobs lying around from the same user.
-                if (driver->my_jobs.count(job_id) > 0) {
-                    if (auto found_status = status_map.find(status);
-                        found_status != status_map.end())
-                        driver->bjobs_cache.insert(
-                            {job_id, found_status->second});
-                    else {
-                        free(line);
-                        fclose(stream);
-                        throw std::runtime_error(
-                            fmt::format("The lsf_status:{} for job:{} was "
-                                        "not recognized\n",
-                                        status, job_id));
+    if (exit_code != 0) {
```

Comment:
We should not consider this a failure, check e.g. https://github.com/equinor/ert/blob/bb9b46bd31a9df05afe7fb12312454b98fe34828/tests/unit_tests/scheduler/test_lsf_driver.py#L412